packages/pyright-internal/src/tests/samples/semantic_highlighting/unknown_any_union.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Callable
+
+
+class A:
+    def __init__(self):
+        self._prop = "yes"
+
+    @property
+    def prop(self):
+        return self._prop
+
+    @prop.setter
+    def prop(self, value):
+        self._prop = value
+
+
+a = A()
+ap = a.prop
+a.prop = "hi"
+
+test_any = Any
+test: Any = s
```

Comment:
maybe have a test using the type alias as a type annotation too
```suggestion
test_any = Any
foo: test_any = 1
test: Any = s
```